chapters/en/module5.md

```diff
@@ -0,0 +1,297 @@
+</opt>
+<opt text='the <code>canadian-languages</code> branch with the <code>app</code> branch'>
+Is <code> canadian-languages</code>  referring to a branch? 
+</opt>
+</choice>
+</exercise>
+
+<exercise id='7' title='When merging branches goes social: opening pull requests'  type='slides, video'>
+<slides source='module5/module5_04' shot='0' start='3:42' end='4:35'> </slides>
+</exercise>
+
+<exercise id='8' title='You are cordially invited to review my changes'>
+
+**True or false**
+
+You should be quite sure when creting a pull request because you will not be able add more commits after you create it.
+
+<choice id = 8>
+<opt text='True' >
+Try open a pull request and pushing new commits and check what happen!
+</opt>
+<opt text='False' correct='true'>
+You can continue adding commits to a pull request pushing to the branch you used to create it. You will see the commits as a timeline when you open it on GitHub. Try it!
+</opt>
+</choice>
+
+**True or false**  
+
+It is possible to open a pull request locally
+
+<choice id = 9>
+<opt text='True' >
+It is possible to <a href="https://docs.github.com/es/enterprise-server@3.1/pull-requests/collaborating-with-pull-requests/reviewing-changes-in-pull-requests/checking-out-pull-requests-locally">check out a pull request locally to solve a merge conflict or test changes before aproving it</a>, but not actually create one.
+</opt>
+<opt text='False' correct='true'>
+You have to enter to your GitHub repository to start a pull request. What you can do locally is to create a new branch and push it to GitHub. Then GitHub will give you the option to start a pull request.
+</opt>
+</choice>
+
+How is it different to open a pull request to merging branches? Is it not basically the same?
+
+Select the correct option
+
+<choice id = 10>
+<opt text='Yes, it is the same, you are merging branches but using GitHub' >
+Even if the final goal of the pull request is to merge your changes with other branch, it is also a way to socialize that you created those changes and also to involve other team members, for example, as reviewers.
+</opt>
+<opt text='You open a pull request to merge branches when you want to involve your collaborators' correct='true'>
+</opt>
+</choice>
+</exercise>
+
+<exercise id='9' title='About the different ways we can collaborate' type='slides, video'>
+<slides source='module5/module5_05' shot='0' start='3:42' end='4:35'> </slides>
+</exercise>
+
+<exercise id='10' title='Between branches and forks'>
+
+In which cases branching would be better that using forks?
+
+<choice id = 11>
+<opt text='The owner of the repository have set branch protection rules to the <code>main</code>  branch'>
+Even if it is a good practice as avoid that contributors push code directly to the <code>main</code> branch by accident, protecting the branch it is not a requirement to collaborate using branching. 
```

Comment:
```suggestion
Even if it is a good practice to avoid that contributors push code directly to the <code>main</code> branch by accident, protecting the branch it is not a requirement to collaborate using branching. 
```